src/Common/src/System/Net/SecurityProtocol.cs

```diff
@@ -15,6 +15,8 @@ internal static class SecurityProtocol
         public const SslProtocols DefaultSecurityProtocols =
             SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
 
+        public const SslProtocols SystemDefaultSecurityProtocols = SslProtocols.None;
```

Comment:
@davidsh I have confirmed that WinHTTP does not support this setting and so we need to rely on the internal configuration, which I left centralized within the `DefaultSecurityProtocols` const.

@stephentoub If possible, I recommend changing the CURL implementation to use internal defaults. This allows applications to benefit from new TLS versions or system-wide changes of the defaults without requiring recompilation.